spec/QueueMiddlewareSpec.php

```diff
@@ -34,7 +34,7 @@ function it_executes_a_command(Producer $producer, QueueableCommand $command)
         $this->execute($command, function() {});
     }
 
-    function it_executes_invokes_the_next_middleware(Producer $producer, Command $command, Middleware $middleware)
+    function it_executes_invokes_the_next_middleware(Producer $producer, stdClass $command, Middleware $middleware)
```

Comment:
Actually, we don't even have to mock an object for this purpose. We are free to instantiate an stdClass. Mocking ways ugly, but this one could work...